tests/BasicSearchTests.FunctionalTests.Core/AutocompleteFunctionalTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using BasicSearchTests.FunctionalTests.Core.Models;
+using BasicSearchTests.FunctionalTests.Core.TestSupport;
+using Xunit;
+using System.Net;
+
+namespace BasicSearchTests.FunctionalTests.Core
+{
+    public class AutocompleteFunctionalTests
+    {
+        private HttpClient _client;
+
+        public AutocompleteFunctionalTests()
+        {
+            // Arrange
+            _client = new HttpClient { BaseAddress = new Uri(EnvironmentSettings.SearchServiceBaseUrl) };
+        }
+
+        [Fact]
+        public async Task CanGetEmptyResult()
+        {
+            // Act
+            var response = await _client.GetAsync(new AutocompleteBuilder { Query = Constants.NonExistentSearchString }.RequestUri);
+            var result = await response.Content.ReadAsAsync<AutocompleteResult>();
```

Comment:
I verified by removing some properties from `AutocompleteResult` and it does in fact deserialize the properties of the object correctly. Any superfluous properties are ignored.